packages/app-store/slackapp/api/commandHandler.ts

```diff
@@ -0,0 +1,26 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+
+import prisma from "@calcom/prisma";
+
+import { showCreateEventMessage, showTodayMessage } from "../lib";
+
+export enum SlackAppCommands {
+  CREATE_EVENT = "create-event",
+  TODAY = "today",
+}
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse) {
+  if (req.method === "POST") {
+    const command = req.body.command.split("/").pop();
+
+    switch (command) {
+      case SlackAppCommands.CREATE_EVENT:
```

Comment:
probably a fetch from user availability here? to remove already booked days/time slots

Reply:
The issue is - the datapicker in slack doesn't allow you to hide times / dates from what I have found